app/views/pages/design.html.erb

```diff
@@ -1,15 +1,11 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-
+buttons design
+<br>
+<br>
 
 <button type="button" class="btn btn-primary btn-action">Primary</button>
-<button type="button" class="btn btn-secondary">Secondary</button>
```

Comment:
Oh why remove the secondary button?

Reply:
If we design as per figma, we don't have secondary button, our 2 buttons are 1: the purple filled-in one, and 2: the purple-outline one. So no need for secondary button, just realised it on saturday
![image](https://user-images.githubusercontent.com/94740205/187127677-c5cd45a1-6373-43d3-979b-20ef8adffac8.png)